src/components/style/modals/dialog.css

```diff
@@ -17,14 +21,21 @@
 }
 
 .dialog-overlay {
+  z-index: var(--zindex-shim);
+  background: var(--color-charcoal);
+  opacity: 0.6;
+}
+
+.dialogfullScreen {
```

Comment:
why not apply the `z-index` and positioning rules directly to `dialog-overlay` and `dialog-wrapper`?

Reply:
@akdetrick the fix is in